### PR TITLE
[transducer] transducer embedding predictor

### PR DESCRIPTION
**Pull request by Mddct**
TODO:
- [x] embedding predictor
- [x] tie embedding
- [x] aishell results using embedding predictor

### CODE REVIEW

File: wenet/transducer/example_embedding_predictor.yaml [hunk 1]
```diff
@@ -0,0 +1,96 @@
+encoder: conformer
```
Comment (robin1001) [inline wenet/transducer/example_embedding_predictor.yaml:1]: please move the file to aishell/rnnt/conf

**Comment by Mddct** [comment]
issues： we should not tie blank when tie embedding ，easy to cause model collapse

**Comment by Mddct** [comment]
> issues： we should not tie blank when tie embedding ，easy to cause model collapse

this is not a problem, due to code bug,  And i'll paste aishell results later which can achieve the same performace as rnn or even better

**Comment by Mddct** [comment]
@robin1001  I remove tie embedding. Maybe I'll re-add it if i find it can improve the accuracy. 
for now:

aishell rnnt-greedy search: 4.892 
160 epoch avg10

I think  it's time to merge it now,

TODO: (next stage)
- [ ]  embedding + conv
- [ ]  this version of embedding + conv

**Comment by robin1001** [comment]
Great job! Let's keep it simple and stupid now. Looking forward to your future work.

**Comment by Mddct** [comment]
WIP， Looks like the conflict's gone, but I need to double check the decoding

**Comment by Mddct** [comment]
> WIP， Looks like the conflict's gone, but I need to double check the decoding

works  ok

**Review by robin1001 — APPROVED** [review]

**Pull request merged**